tensorflow/contrib/framework/BUILD

```diff
@@ -24,7 +27,42 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
```

Comment:
Can you try rebasing with master? Hopefully these were transient errors and are fixed now :-/